src/ctia/flows/hooks/event_hooks.clj

```diff
@@ -61,9 +64,27 @@
     event))
 
 (defn- judgement?
-  [{t :type :as event}]
+  [{{t :type} :entity :as event}]
   (= "judgement" t))
 
+(def judgement-prefix "judgement-")
+
+(defn starts-with?
+  "Added to clojure.string in Clojure 1.8"
+  [^String s ^String ss]
+  (and s (.startsWith s ss)))
+
+(s/defn realize-verdict-wrapper :- vs/StoredVerdict
```

Comment:
This is a temporary workaround due to not being able to retrieve a verdict by judgement ID. That will be implemented in #354. Once that is done, this reuse of judgement IDs to build the verdict ID will not be necessary, and realize-verdict can be called directly.